.allure/allure-2.8.1/bin/allure

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env sh
```

Comment:
Please add `.allure` folder to `.gitignore` and remove it from remote repository

Reply:
Removed from repository and .gitignore is refactored.